projects/canopy/src/lib/quick-action/quick-action.component.scss

```diff
@@ -12,7 +12,7 @@
   border: 0;
   background-color: transparent;
   appearance: none;
-  color: var(--link-color);
+  color: var(--quick-action-color);
```

Comment:
Aren't these the same colour though?

```
:root {
  --quick-action-color: var(--link-color);
```

Reply:
Do we need a different variable in this case?

Reply:
I see what you mean, the extra variable could allow us to change the colour in the future (but [YAGNI](https://martinfowler.com/bliki/Yagni.html)). Annoyingly in our [semver rules it would be a breaking change](https://github.com/Legal-and-General/canopy/blob/master/docs/BREAKING_CHANGES.md) to remove that variable.

Reply:
I came across a use-case when applying a syndicate/theme. I wanted to change the Quick Action colour, but not necessarily make a global change to the `--link-color` at this stage, so it feels useful to be able to get per-component control if needed. It's also a pattern that has appeared in other components, whereby component-specific vars are just referencing global vars. So to me if feels worthwhile, happy to discuss in more detail though.

Reply:
I see your point @owensgit but assigning it straight to the `--link-color` will make developing a syndicated site a lot less hassle. Is your case only due to the fact that you don't want to change the other links globally?

Reply:
Should the quick action be a different colour to the global link color though? Shouldn't it just be the case that you can override the `--link-color`

Reply:
Thanks @elenagarrone @paul-request  - yes I'd usually just override `--link-color` but in this particular case I was looking to make an incremental change and reduce the scope of testing. Given that there was already a `--quick-action-color` I thought perhaps I could just change that, but then realised it wasn't hooked up. I also have seen that elsewhere in Canopy we have these component specific variables which reference global ones e.g. `--quick-action-color: var(--link-color)`. 

I do see what you mean though, do you think we should reduce our component specific variables in favour of using the global ones more directly in our components? Perhaps this warrants a proper discussion?

Reply:
Yes, I think that would be better going forward 🤔

Reply:
@andy-polhill  - just wondering if it would really be a breaking change to remove that variable? It's never actually been hooked up in the CSS, so it's effectively been doing nothing. So removing it would technically not break anything 🤔

Reply:
oh I think in that case it should be fine, it's just a `fix`

Reply:
@andy-polhill @elenagarrone @paul-request  - thanks for the above discussion, I've now removed the specific variable `--quick-action-link-color` completely - the commit now reads:

```
fix(quick-action): remove specific color variable

removes the specific color variable for the quick action, in favour of
using the global --link-color instead
```

Please take another look when you can :)